src/Shells/StrixMusic.Shells.ZuneDesktop/Controls/Views/Collection/ZuneTrackCollectionItem.cs

```diff
@@ -44,15 +52,13 @@ async Task ExecuteAsync(TrackViewModel? newValue)
             {
                 if (Track is not null)
                 {
-                    DetachEvents(Track);
                     ArtistNamesMetadata.Clear();
                 }
 
                 if (newValue is not null)
                 {
-                    AttachEvents(newValue);
```

Comment:
Since we're no longer listening to `ArtistItemsChanged`, the `ArtistNamesMetadata` won't be updated properly if changed in the backend.

Committing as-is would cause a regression. Let's switch to Behaviors and do this in XAML instead, that way it can still listen for changes while also respecting virtualization (this should avoid runaway memory usage).